core/src/com/unciv/logic/civilization/diplomacy/DiplomaticStatus.kt

```diff
@@ -5,5 +5,7 @@ import com.unciv.logic.IsPartOfGameInfoSerialization
 enum class DiplomaticStatus : IsPartOfGameInfoSerialization {
     Peace,
     Protector,  //city state's diplomacy for major civ can be marked as Protector, not vice versa.
-    War
+    DefensivePact,
+    OffensiveWar,
```

Comment:
We have to retain War, since older games use it and this is a breaking change

Reply:
Should we replace all current Wars with DefensiveWars in some function once the game is loaded or add it to IsAtWar()?
If we really want to, we can remove OffensiveWar and DefensiveWar since signing a defensive pact no longer affects wars started in the past.

Reply:
Is it doesn't affect it, then yes, let's revert this. It will save us a huge headache